armsrc/Standalone/Makefile.inc

```diff
@@ -57,6 +41,10 @@ endif
 ifneq (,$(findstring WITH_STANDALONE_HF_14ASNIFF,$(APP_CFLAGS)))
     SRC_STANDALONE = hf_14asniff.c
 endif
+# WITH_STANDALONE_HF_15693SNIFF
+ifneq (,$(findstring WITH_STANDALONE_HF_15693SNIFF,$(APP_CFLAGS)))
+    SRC_STANDALONE = hf_15693sniff.c
```

Comment:
Please rename the file itself as hf_15sniff.c

Reply:
Done